freqtrade/persistence.py

```diff
@@ -66,25 +66,45 @@ def has_column(columns, searchname: str) -> bool:
     return len(list(filter(lambda x: x["name"] == searchname, columns))) == 1
 
 
+def get_column_def(columns, column: str, default: str) -> str:
+    return default if not has_column(columns, column) else column
+
+
 def check_migrate(engine) -> None:
     """
     Checks if migration is necessary and migrates if necessary
     """
     inspector = inspect(engine)
 
     cols = inspector.get_columns('trades')
+    tabs = inspector.get_table_names()
+    table_back_name = 'trades_bak'
+    i = 0
+    while table_back_name in tabs:
```

Comment:
We could use `enumerate` here, which seems more pythonic, e.g.:
```
for i, table_back_name in enumerate(tabs):
    ...
```

Reply:
beeing honest, i didnt expect this to work when `tabs` is updated within the for loop - but it does indeed work.